src/test/README.md

```diff
@@ -0,0 +1,34 @@
+# Testing in Wildbook
+
+## JUnit for Unit Testing
+
+Wildbook uses [JUnit 5](https://junit.org/junit5/docs/current/user-guide/) for unit testing. All unit tests _must run and pass_ in order for changes to be
+incorporated into the Wildbook codebase. Tests should be run automatically with maven when executing `mvn clean install`.
+
+**All new code** should have appropriate unit tests written to validate all components of the new code are working as expected. Take care to also test
+invalid data cases and failure/exception cases.
+
+Tests should be created under the `src/test/java/org/ecocean/` directories, corresponding to the java class which are testing. The test class name should follow
+the convention of adding the suffix `Test` to the java class name, such as `AnnotationTest.java` for tests of code within `Annotation.java`.
+
+Ultimately, almost all Wildbook java classes will have a corresponding `*Test.java` class. If one does not exist to add new tests to, it should be created
```

Comment:
~Ultimately~ Eventually

"That being said, edits to existing classes may not require full understanding of a class, while testing does. If you choose not to make a unit test for an existing class, make a note of that choice in the PR and explain why."